### PR TITLE
Show correct Homebrew directory for 1.0.0

### PR DESCRIPTION
**Pull request by bradley-curran**
Homebrew's default directory has changed to /usr/local/Homebrew in 1.0.0. 

The homepage has been updated for all languages. 

Let me know if I need to make any changes for the PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 pagecontent:
   question: What Does Homebrew Do?
   what: Homebrew installs <a href="https://github.com/Homebrew/homebrew-core/tree/master/Formula" title="List of Homebrew packages">the stuff you need</a> that Apple didn’t.
-  how: Homebrew installs packages to their own directory and then symlinks their files into <code>/usr/local</code>.
+  how: Homebrew installs packages to their own directory and then symlinks their files into <code>/usr/local/Homebrew</code>.
```
Comment (MikeMcQuaid) [inline index.html]: This isn't correct, they are still symlinked into `/usr/local`.

File: _layouts/index.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h2 id="question">{{ page.pagecontent.question }}</h2>
         <div class="col-2">
 {% highlight bash %}
 $ brew create https://foo.com/bar-1.0.tgz
-Created /usr/local/Library/Formula/bar.rb
+Created /usr/local/Homebrew/Library/Formula/bar.rb
```
Comment (MikeMcQuaid) [inline _layouts/index.html]: Technically `/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/bar.rb`

Comment (bradley-curran) [inline _layouts/index.html]: I'll create a new PR based on these changes.

Comment (MikeMcQuaid) [inline _layouts/index.html]: 🆒. In future we'd generally prefer updating the same PR, though.

**Pull request closed without merging**